Healthy/Classes/ReusableViews/FormTextField/FormTextField.swift

```diff
@@ -0,0 +1,90 @@
+
+import UIKit
+
+class FormTextField: UIView {
+    
+    // MARK: Outlets
+    
+    @IBOutlet private weak var contentView: UIView!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var inputTextField: UITextField!
+    @IBOutlet private weak var errorLabel: UILabel!
+    
+    // MARK: Properties
+    
+    private var onChange: (String) -> () =  { _ in }
+    
+    var title: String = "" {
+        didSet {
+            self.titleLabel.text = title
+        }
+    }
+    
+    var placeholder: String = "" {
+        didSet {
+            self.inputTextField.placeholder = placeholder
+        }
+    }
+    
+    var keyboardType: UIKeyboardType = .default {
+        didSet {
+            self.inputTextField.keyboardType = keyboardType
+        }
+    }
+    
+    // MARK: Init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        initView()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        initView()
+    }
+    
+    private func initView() {
+        Bundle.main.loadNibNamed(FormTextField.reusableIdentifier, owner: self, options: nil)
+        addSubview(contentView)
+        contentView.frame = self.bounds
+        contentView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+        configureLayout()
+        bindTextFieldChanges()
+    }
+}
+
+// MARK: Internal Handlers
+
+extension FormTextField {
+    func setError(_ errorText: String) {
+        self.errorLabel.text = ""
+        self.errorLabel.text = errorText
+    }
```

Comment:
```suggestion
    func setError(_ errorText: String?) {
        self.errorLabel.text = errorText
    }
```

This should be computed property as well.